examples/lvgldemo/lvgldemo.c

```diff
@@ -175,6 +180,10 @@ int main(int argc, FAR char *argv[])
   const int func_key_pair_len = sizeof(func_key_pair) /
```

Comment:
Note for the future: `nitems(func_key_pair);` can be used instead

Reply:
Done
https://github.com/apache/nuttx-apps/pull/1587